File: src/api/rule/book.js
module.exports = {
	'id|+1':1,
    'isbn':'ISBN' + '@character("upper")',
    'name':'@character("upper")',
    'type':function(){
    	var types = ['少儿','青年'];
    	var rnd = Math.floor(Math.random()*types.length)
    	return types[rnd]
    }(),
    'publisher':'少儿出版社',
    'author':'@character("upper")',
    'intro':'说明说明说明',
    'price':'@natural(1, 1000)', //大于等于零的整数
    'publishdate':'2020/10/19',
    'pic':'',
    'count':'@natural(1, 1000)', //大于等于零的整数
    'isnew':true,
    'isrecommend':true,
    'isdeleted':false
};

File: java/bookshop/src/model/Book.java
package model;

public class Book {
	public int ID;
	public String isbn;
	// 图书名称
	private String name;
	// 图书价格
	public float price;
	// 图书作者
	public String author;
	// 出版社
	public String publisher;
	// 图书分类号
	public String type;
	// 图书介绍
	public String intro;
	// 图书发版日期
	public String publishdate;
	// 图书图片信息
	public String pic; 
	/**
	 * 获取图书名称
	 */
	public String getBookName() {
		return this.name;
	}
	
	public void setBookName(String name) {
		this.name = name;
	}
}

